discord/commands/core.py

```diff
@@ -1119,6 +1118,8 @@ def __init__(
         description: str | None = None,
         guild_ids: list[int] | None = None,
         parent: SlashCommandGroup | None = None,
+        cooldown: Any = None,
+        max_concurrency: Any = None,
```

Comment:
```suggestion
        cooldown: CooldownMapping | None = None,
        max_concurrency: MaxConcurrency | None = None,
```
Also, you need to import them in TYPE_CHECKING

Reply:
Oh right. I'll do this tomorrow